torch_uncertainty/models/resnet/std.py

```diff
@@ -197,11 +199,15 @@ def __init__(
         dropout_rate: float,
         groups: int,
         style: str = "imagenet",
+        num_estimators: int = None,
```

Comment:
I agree it's better to keep num_estimators as a parameter name to avoid increasing their number, but it may sound strange.

Reply:
I've added a docstring to make it a bit more explicit.